Source/Blazorise/RadioGroup.razor.cs

```diff
@@ -79,12 +79,12 @@ internal async Task NotifyRadioChanged( Radio<TValue> radio )
         /// <inheritdoc/>
         public override async Task SetParametersAsync( ParameterView parameters )
         {
-            await base.SetParametersAsync( parameters );
-
             if ( parameters.TryGetValue<TValue>( nameof( CheckedValue ), out var result ) )
             {
-                await CurrentValueHandler( result?.ToString() );
```

Comment:
`CurrentValueHandler `must be here. Without it validation will not work.

Reply:
Hmm, OK. Validations don't currently work for RadioGroups anyways. #1231 But I guess it's better than making it worse. (I did have suggested fixes in the issue.)

Reply:
I have put it back and did it another way in the next commit.